avalanche/spec.md

```diff
@@ -173,6 +184,9 @@ Assuming there are not any `no` votes then we expect it to take 134 queries to f
 take a minimum of 1.34 seconds to send off enough requests to finalize the transaction. Add network latency on top of that and we expect most transactions
 to finalize in 2 to 3 seconds.
 
+> So 134 round trip messages?  Given that transactions currently need 1 round trip, this will have a major effect on scalability.
```

Comment:
It is 134 round trips, however they are batched. So if, as an unlikely example, a node received 1000 txs at the exact same time it would still only make 134 round trips (with each round trip containing 1000 txids).

Reply:
I understand they are batched but that's a lot, will have a major effect on scalability.  Especially since the requests are centralized to miners of the last N blocks.

Here's an idea: allow a node to forward the votes it received from multiple nodes to other nodes. This will at least decentralize...